src/Symfony/Component/HttpClient/Tests/AmpHttpClientTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient\Tests;
+
+use Symfony\Component\HttpClient\AmpHttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class AmpHttpClientTest extends HttpClientTestCase
+{
+    protected function getHttpClient(string $testCase): HttpClientInterface
+    {
+        return new AmpHttpClient(['verify_peer' => false, 'verify_host' => false]);
+    }
+
+    public function testProxy()
+    {
+        $this->markTestSkipped('A real proxy server would be needed.');
```

Comment:
You could use https://github.com/leproxy/leproxy for that.

Reply:
I could... :)